source/includes/steps-fle-convert-to-a-remote-master-key-aws.yaml

```diff
@@ -255,6 +303,9 @@ content: |
 title: Update the Automatic Encryption JSON Schema
 ref: update-the-json-schema
 content: |
-  If you embedded the key id of your data encryption key in your
-  automatic encryption rules, you will need to update the :ref:`JSON
-  Schema <fle-define-a-json-schema>` with the new data encryption key id.
+  If you previously embedded the key id of your data encryption key in your
```

Comment:
[nit] according to the Style Guide, we should use ID rather than id.

Reply:
Sounds good to me, will update!